mlir/test/Dialect/ArmSME/vector-legalization.mlir

```diff
@@ -266,3 +266,39 @@ func.func @transpose_f32_scalable_4x16_via_write(%src: memref<?x?xf32>, %dest: m
   vector.transfer_write %0, %dest[%c0, %c0] {permutation_map = #transpose, in_bounds = [true, true]} : vector<[4]x[16]xf32>, memref<?x?xf32>
   return
 }
+
+// -----
+
+// CHECK-LABEL: @extract_from_vector_create_mask_non_constant_dim(
+// CHECK-SAME:                                                    %[[DIM0:[a-z0-9]+]]: index,
```

Comment:
wondering why to use a tighter regex instead of `.*`, I've hit this issue with CHECK-SAME on args several times where it'll only match if all the args are on the same CHECK-SAME line, is this because of `.*`?

Reply:
(just a question btw, am curious, not asking for any changes)

Reply:
Yeah, with `.*` it sometimes matches too much, but using a tighter regex like `[a-z0-9]+` avoids that (that's why I do that here).